_rules/links-with-identical-names-and-context-serve-equivalent-purpose-fd3a94.md

```diff
@@ -220,7 +221,7 @@ These two SVG `a` and HTML `a` elements have the same [accessible name][], same
 
 #### Passed Example 9
 
-These two HTML `a` elements have the same [accessible name][] and [context][programmatically determined link context], but resolve to different resources. However, the purpose of these links is [ambiguous to users in general](https://www.w3.org/TR/WCAG22/#dfn-ambiguous-to-users-in-general). Thus all readers are unsure about the destination and the person with a disability is not at any disadvantage.
+These two HTML `a` elements have the same [accessible name][], the same [accessible description][], and the same [context][programmatically determined link context], but go to different resources. However, there is no visual information to let users know that both links resolve to non-equivalent resources. 
```

Comment:
Starting with this example, we began using "the same accessible description", whereas in previous cases we used "empty accessible description". I recommend maintaining consistency throughout. It might also be helpful to include examples that uses title or aria-describedby attributes both in passed and failed for comparison.